src/main/java/de/tum/in/www1/artemis/domain/Course.java

```diff
@@ -150,6 +151,13 @@
     @JsonIgnoreProperties("course")
     private Set<Exam> exams = new HashSet<>();
 
+    @ManyToMany
```

Comment:
please talk to @stefanwaldhauser about the use of `mappedBy`

Reply:
Added **mappedBy** with e4c6beb